tests/dependencies_test.go

```diff
@@ -31,6 +31,8 @@ func TestDependenciesLifecycle(t *testing.T) {
 	invokeWordpressBundle(p, namespace)
 
 	uninstallWordpressBundle(p, namespace)
+
+	noReallyUninstallWordpressBundle(p, namespace)
```

Comment:
😂

Reply:
Oh, I just realized, should I just place this uninstall+delete logic into `cleanupWordpressBundle`?  Or keep it separate?

Reply:
(I updated `cleanupWordpressBundle` w/ uninstall+delete logic in https://github.com/deislabs/porter/pull/1189/commits/6af718a9446dbd9c99821092c06f89822c553dde)